scripts/github_pr/interactions/ia_cloud.rb

```diff
@@ -1,13 +1,20 @@
 # Github_PR interactions (filter and action classes)
+require "open-uri"
 
 module GithubPR
   class JenkinsJobTriggerAction < RunCommandAction
+    JENKINS_URL="https://ci.suse.de"
+
     def action(pull)
       base_cmd = command("job_cmd")
       job_parameters(pull).each do |build_mode, job_paras|
         one_cmd = base_cmd + [@c["job_name"]] + job_paras
-        system(*one_cmd) or raise
         puts "Triggering jenkins job for PR #{pull.number} in #{build_mode} mode"
+        if @c.has_key?("log_rebuild_link") && @c["log_rebuild_link"] then
+          puts "== Link to _rebuild_ the job that was just triggered (NOTE: make sure there are no identical parallel jobs):"
+          puts "== #{JENKINS_URL}/job/#{@c["job_name"]}/parambuild/?#{job_paras.map{|kv| URI::encode(kv)}.join("&")}"
```

Comment:
URI::encode seems to not encode some special chars like `=` and `&` 
those can be specified as a 2nd arg - see https://ruby-doc.org/stdlib-1.9.3/libdoc/uri/rdoc/URI/Escape.html